packages/app/src/contacts/data-structures/contacts-hash-table.structure.ts

```diff
@@ -5,14 +5,44 @@
 
 import { HashTable } from "App/core/data-structures"
 import { Contact } from "App/contacts/dto"
+import { intl } from "App/__deprecated__/renderer/utils/intl"
 
 export class ContactsHashTable extends HashTable<Contact> {
+  private favouriteKey = intl.formatMessage({
```

Comment:
Feel free to disagree, but TBH I don't feel that `ContactsHashTable` is a good place for handling `favourites`. 
1. This is a data structure so it should be quite simple and IMHO now it handles more than the name suggests.
2. At least for me, using translations in data structure is not a perfect solution. Translations seems to belong more to visual layer.

I see your point (to have all contacts grouped in one place and later just to display them in an easy way), but maybe a better solution would be to leave `ContactsHashTable` as it was and to store/extract favourites in a different way (e.g. additional property for `favourites` in `ContactsHashTable`? Additional structure that would contain `ContactsHashTable` and `favourites`?). As I said, feel free to disagree :)

Reply:
https://github.com/mudita/mudita-center/pull/882/commits/d60326ede70850be3debdf3471118d62c1390a5d